geode/github-repo

```diff
@@ -0,0 +1 @@
+https://github.com/apache/geode.git
```

Comment:
This should _not_ end in `.git` (again, see other folders for examples).  Also, this is going to result in links that include https://github.com/apache/geode/issues (which are disabled on https://github.com/apache/geode), so we'll likely need at _least_ `geode/issues.md` as well to instruct users where to file issues.

Again, see any of the other folders for examples of how this ought to be formatted:
```console
$ ls */issues.md
aerospike/issues.md    fedora/issues.md          r-base/issues.md
amazonlinux/issues.md  ibmjava/issues.md         ubuntu/issues.md
centos/issues.md       php-zendserver/issues.md  xwiki/issues.md
```